### PR TITLE
feat(spinner): set shim styles as default

### PR DESCRIPTION
**Pull request by mivaylo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by valentin-mladenov** [comment]
After PR #992 is merged in `next` branch and this PR is synced with it. 
Changes introduced in commit c2568ed must be reverted and spinner must use proper 18x18 values.

**Review by valentin-mladenov — COMMENTED** [review]
Check my comments.

File: projects/angular/src/progress/spinner/_spinner.clarity.scss [hunk 1]
```diff
@@ -27,138 +27,69 @@
...
       &::after {
-        @include mixins.css-var(
-          border-color,
-          clr-spinner-fill-color,
-          spinner-variables.$clr-spinner-color,
-          variables.$clr-use-custom-properties
-        );
+        border-color: spinner-variables.$clr-spinner-fill-color;
+        border-right-color: transparent;
+        border-bottom-color: transparent;
```
[the remark this answers is not in the document]
Comment (mivaylo) [reply projects/angular/src/progress/spinner/_spinner.clarity.scss:92]: border-color: spinner-variables.$clr-spinner-fill-color;

on line 90 is actually overwriting them, so we do need them

File: projects/angular/src/progress/spinner/_spinner.clarity.scss [hunk 2]
```diff
@@ -27,138 +27,69 @@
...
       &::after {
-        @include mixins.css-var(
-          border-color,
-          clr-spinner-fill-inverse-color,
-          spinner-variables.$clr-spinner-inverse-color,
-          variables.$clr-use-custom-properties
-        );
+        border-color: spinner-variables.$clr-spinner-fill-inverse-color;
+        border-right-color: transparent;
+        border-bottom-color: transparent;
```
[the remark this answers is not in the document]
Comment (mivaylo) [reply projects/angular/src/progress/spinner/_spinner.clarity.scss:80]: this line is overwriting the border colors "border-color: spinner-variables.$clr-spinner-fill-inverse-color;" so we do need the transparent rules

File: projects/demo/src/app/buttons/button-loading.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ export class ButtonLoadingDemo {
   disabledStateDisabled = false;
   enabledStateDisabled = false;
 
-  validateSmState = false;
+  validateSmState = true;
```
[the remark this answers is not in the document]
Comment (mivaylo) [reply projects/demo/src/app/buttons/button-loading.ts:23]: Sorry :)

File: projects/angular/src/utils/_overwrites.clarity.scss [hunk 4]
```diff
@@ -790,9 +790,6 @@ $clr-progress-bgColor: null; // $clr-color-neutral-200
 /*================================================================================*/
 
 $clr-spinner-color: null; // $clr-color-action-600
-$clr-spinner-bg-color: null; // $clr-color-neutral-1000
-$clr-spinner-inverse-bg-color: null; // $clr-color-neutral-0
-$clr-spinner-opacity: null; // 0.15
```
[the remark this answers is not in the document]
Comment (mivaylo) [reply projects/angular/src/utils/_overwrites.clarity.scss:795]: We created a task to delete the whole file

[the remark this answers is not in the document]
Comment (mivaylo) [reply projects/angular/src/utils/_overwrites.clarity.scss:795]: It seems to be not needed anymore

**Review by valentin-mladenov — APPROVED** [review]
LGTM

**Pull request merged**